### PR TITLE
Fix deployment bug

### LINKED ISSUE #55: Look into deployment issues w/ manifest.js file
Currently we gitignore the manifest.js file. 

Locally, we have to have this file and also uncomment `require "sprockets/railtie"` in applicaiton.rb file in order to run graphiql locally. 

This ticket involves digging into this issue more to avoid this workaround.

### PR DESCRIPTION
**Pull request by leahriffell**
### Code Highlights:
* Let's see if this works 🤞🏼 
* Explicitly clear precompiling of assets in Rakefile
* I don't fully understand this issue but I believe it is an issue with a Rails 6 API and Sprockets 4

### Where should the reviewer start?
 *  Rakefile

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
* If this works, I will update the README instructions in a separate PR

### Issues:
* Addresses #55 

### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations:

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -4,3 +4,6 @@
 require_relative "config/application"
 
 Rails.application.load_tasks
+
+Rake::Task["assets:precompile"].clear
```
Comment (LHJE) [inline Rakefile:8]: NICE!

**Review by LHJE — APPROVED** [review]
Nicely done!

File: Rakefile (hunk 1 above)
Comment (dhalverson) [reply Rakefile:8]: Let's chat about this one next time we meet!

**Review by dhalverson — APPROVED** [review]
Looks good, I want to discuss quickly next time we meet on this one please

**Review by AngelaGuardia — APPROVED** [review]
Nice solution @leahriffell

**Pull request merged**